src/cadenza.js

```diff
@@ -442,7 +448,10 @@ export class CadenzaClient {
    * _Note:_ The file name, if not provided, is generated from the name of the workbook view and the current date.
    *
    * @param {WorkbookViewSource} source - The workbook view to download data from
-   * @param {string} mediaType - The media type to use for the data
+   * @param {string} mediaType - The media type to use for the data. Allowed are
+   * * 'text/csv'
+   * * 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+   * * 'application/pdf' (for Jasper Report views)
```

Comment:
Good to mention them here :+1:

Proposal: Put "(Excel)" behind the long one.

Reply:
Good idea, will do.